docker/services/devnet.yml

```
@@ -1,15 +1,14 @@
-    container_name: devnet
-    image: ghcr.io/ensdomains/contracts-v2:main-e8696c6
+    image: ghcr.io/ensdomains/contracts-v2:main-580c60a
     command: ./script/runDevnet.ts --testNames
     pull_policy: always
     ports:
       - "8545:8545"
     environment:
       ANVIL_IP_ADDR: "0.0.0.0"
     healthcheck:
-      test: ["CMD", "curl", "--fail", "-s", "http://localhost:8000/health"]
+      test: [ "CMD", "curl", "--fail", "-s", "http://localhost:8000/health" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)